tools/integration_tests/run_e2e_tests.sh

```diff
@@ -50,6 +50,18 @@ if [[ $# -ge 6 ]] ; then
   fi
 fi
 
+if ${RUN_TESTS_WITH_ZONAL_BUCKET}; then
+  if ${RUN_TESTS_WITH_PRESUBMIT_FLAG}; then
```

Comment:
Why do we need this if? We are anyway passing bucket-location compulsory for this script.